recipes/libwebp/all/patches/0002-qnx.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/cmake/deps.cmake b/cmake/deps.cmake
+index f19c0378..7b2d7cd7 100644
+--- a/cmake/deps.cmake
++++ b/cmake/deps.cmake
+@@ -24,7 +24,8 @@ check_c_source_compiles("
+ # Check for libraries.
+ find_package(Threads)
+ if(Threads_FOUND)
+-  if(CMAKE_USE_PTHREADS_INIT)
++  # work around cmake bug on QNX (https://cmake.org/Bug/view.php?id=11333)
```

Comment:
```suggestion
+  # work around cmake bug on QNX (https://chromium-review.googlesource.com/c/webm/libwebp/+/2637274)
```
this cmake issue is useful to understand the bug, unfortunately it does not give the information of which version of libwbp already have this fix. deployment of future versions will be easier if the patch links to the upstream issue directly.

Reply:
Thats all I got, the CMake issue also fixes the QNX, read the gerrit thread if you are interested